python/nav/web/sass/nav/custom.scss

```diff
@@ -143,7 +143,7 @@ fieldset legend {
 
 /* Sudoer bar is the bar that appears when you switch users from the
    'User Administration' page */
-.info-bar {
+.sudoer-bar {
```

Comment:
Is this accidentally in here? This reverts a change I added in #3408

Reply:
It is not! I reverted the file to master (or at least tried), as I put the driver css in overrides instead